pkg/components/components.go

```diff
@@ -10,10 +10,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const Registry = "quay.io"
 const Name = "kubevirt-web-ui-operator"
 
 func GetDeployment(namespace string, repository string, tag string, imagePullPolicy string) *appsv1.Deployment {
-	image := fmt.Sprintf("%s/%s:%s", repository, Name, tag)
+	image := fmt.Sprintf("%s/%s/%s:%s", Registry, repository, Name, tag)
```

Comment:
It would be nightmare to build this for downstream with these hardcoded values.
The caller code needs to provide correct value when calling the `GetDeployment`.
I will push a fix to HCO.